docs/ssms/release-notes-ssms.md

```diff
@@ -89,7 +89,7 @@ SSMS 18.4 is the latest general availability (GA) release of SSMS. If you need a
 
 * Database Diagram created from SSMS running on machine A cannot be modified from machine B (SSMS crashes). See [UserVoice](https://feedback.azure.com/forums/908035/suggestions/37992649) for more details.
 
-* There are redraw issues when switching between multiple query windows. See UserVoice for more details. A workaround for this issue is to disable hardware acceleration under *Tools > Options*.
+* There are redraw issues when switching between multiple query windows. See [UserVoice](https://feedback.azure.com/forums/908035/suggestions/37474042) for more details. A workaround for this issue is to disable hardware acceleration under *Tools > Options*.
```

Comment:
Please change the link to - https://feedback.azure.com/forums/908035-sql-server.

Reply:
@SQLGB